raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -1171,7 +1171,10 @@ private DescribeQuorumResponseData handleDescribeQuorumRequest(
         }
 
         if (!quorum.isLeader()) {
-            return DescribeQuorumRequest.getTopLevelErrorResponse(Errors.INVALID_REQUEST);
+            return DescribeQuorumResponse.singletonErrorResponse(
```

Comment:
We have documentation comments for `handleFetchRequest` and other `handleXXXRequest` methods, how about adding documentation comments for `handleDescribeQuorumRequest`.

Reply:
Hmm, this part at least seems self-explanatory. Do you see something else that deserves more explanation? Would you mind submitting a follow-up PR if you do? I am happy to review.